### DISCUSSION TITLE
Unable to Pull PCAP

### DISCUSSION THREAD

**Discussion by out-n-about** [Security-Onion-Solutions/securityonion#1974] [Unsupported Versions] [answered]
Hello, 

Recently installed SO 2.3.2 in a stand alone deployment.  I know that it is capturing network traffic and PCAP is available, however I am unable to pull specific PCAP out.

Meaning I can get PCAP if I only specify Begin date/time, End date/time and sensor.  But that gives everything in that time period.

If I specify SRC/DST IP/Port and date/time and sensor SO returns nothing.

If I click on the PCAP pivot link from alerts it also returns nothing.  

Is there something I need to reconfigure, restart?

thanks

**Comment by dougburks** [comment]
Anything special about the traffic you're monitoring?  Any kind of encapsulation like GRE, VXLAN, etc?

**Comment by out-n-about** [comment]
It is deployed into AWS.  The alerts and Zeek contain the right IP
addresses however when I try to search for PCAP for a specific IP SO
returns nothing.  If I only search for PCAP for a date/time period I get
results.

**Comment by weslambert** [comment]
This is because the traffic is transferred in a tunnel (VXLAN encapsulated, so it would be the IPs of the VXLAN source/dest, not the inner details).  We've implemented partial support for VXLAN decapsulation for PCAP retrieval, but it is not yet fully supported.

**Comment by weslambert** [comment]
Have you tried pulling for different types of traffic (zeek conn, vs http vs alert) to see if the results are any different?

**Comment by dougburks** [comment, marked as answer]
As Wes mentioned, we don't fully support VXLAN decapsulation for pcap retrieval. We are tracking this as issue https://github.com/Security-Onion-Solutions/securityonion/issues/773.
